graphsdk/src/androidTest/java/com/microsoft/graph/logger/MockLogger.java

```diff
@@ -22,28 +22,62 @@
 
 package com.microsoft.graph.logger;
 
+import java.util.ArrayList;
+
 /**
  * Mock {@see ILogger}, logs nothing
  */
 public class MockLogger implements ILogger {
 
+    private LoggerLevel mLevel = LoggerLevel.Error;
+    private ArrayList<String> mLogMessage;
+
+    public MockLogger(){
+        if(mLogMessage == null){
+            mLogMessage = new ArrayList<String>();
+        }
+    }
+
     @Override
     public void setLoggingLevel(final LoggerLevel level) {
-
+        mLevel = level;
     }
 
     @Override
     public LoggerLevel getLoggingLevel() {
-        return null;
+        return mLevel;
     }
 
     @Override
     public void logDebug(final String message) {
-
+        mLogMessage.add(message);
     }
 
     @Override
     public void logError(final String message, final Throwable throwable) {
+        mLogMessage.add(message);
+    }
+
+    public ArrayList<String> getLogMessages() {
+        return mLogMessage;
+    }
+
+    public int getSizeByContent(final String message){
```

Comment:
This function is unused, I'm also not clear on the purpose, it seems to return the count of the number of matching messages.